tests/test_validator.py

```diff
@@ -69,7 +69,7 @@ def test_global_namespace_variables():
 
 
 def test_local_namespace_variables():
-    IntList = List[int]
+    IntList = List[int]  # noqa: F841
```

Comment:
I had to rebuild the lockfile instead of refreshing it for some reason. Not a big deal but that updated Ruff, which triggered this false positive.